SDK_versioned_docs/version-3.0.0/subgraph-example.md

```diff
@@ -0,0 +1,282 @@
+
+```
+query pools( $skip: Int!) {
+    pools(
+      first: 1000
+      skip: $skip
+      orderDirection: asc
+    ) {
+      id
+      sqrtPrice
+      token0 {
+        id
+      }
+    	token1{
+      id
+    }
+    }
+  }
+
+```
+
+### Most Liquid Pools
+Retrieve the top 1000 most liquid pools. You can use this similar set up to orderBy other variables like number of swaps or volume.
+
+```
+{
+ pools(first: 1000, orderBy: liquidity, orderDirection: desc) {
+   id
+ }
+}
+```
+
+### Pool Daily Aggregated
+This query returns daily aggregated data for the first 10 days since the given timestamp for the UNI-ETH pool.
+
+```
+{
+  poolDayDatas(first: 10, orderBy: date, where: {
+    pool: "0x1d42064fc4beb5f8aaf85f4617ae8b3b5b8bd801",
+    date_gt: 1633642435
+  } ) {
+    date
+    liquidity
+    sqrtPrice
+    token0Price
+    token1Price
+    volumeToken0
+    volumeToken1
+  }
+}
+```
+
+## Swap Data
+
+### General Swap Data
+To query data about a particular swap, input the transaction hash + "#" + the index in the swaps the transaction array.
+Reference for the full [swap schema](https://github.com/Uniswap/v3-subgraph/blob/main/schema.graphql#L353).
+
+This query fetches data about the sender, receiver, amounts, transaction data, and timestamp for a particular swap.
+```
+{
+   swap(id: "0x000007e1111cbd97f74cfc6eea2879a5b02020f26960ac06f4af0f9395372b64#66785") {
+    sender
+    recipient
+    amount0
+    amount1
+    transaction {
+      id
+      blockNumber
+      gasUsed
+      gasPrice
+    }
+    timestamp
+    token0 {
+      id
+      symbol
+    }
+    token1 {
+      id
+      symbol
+    }
+   }
+ }
+ ```
+
+### Recent Swaps Within a Pool
+You can also filter swap data by a pool address by setting the `where` field. This example fetches data about multiple swaps for the USDC-USDT pool, ordered by timestamp.
+
+```
+{
+swaps(orderBy: timestamp, orderDirection: desc, where:
+ { pool: "0x7858e59e0c01ea06df3af3d20ac7b0003275d4bf" }
+) {
+  pool {
+    token0 {
+      id
+      symbol
+    }
+    token1 {
+      id
+      symbol
+    }
+  }
+  sender
+  recipient
+  amount0
+  amount1
+ }
+}
+```
+## Token Data
+
+Fetch token data by inputting the token contract address. Any token that is apart of a Uniswap pool can be queried and the data that is returned is aggregated across all the pools the token is apart of.
+
+### General Token Data
+
+This queries the decimals, symbol, name, pool count, and volume in USD for the UNI token. Reference the full [token schema](https://github.com/Uniswap/v3-subgraph/blob/main/schema.graphql#L38) for all possible fields you can query.
+
+
+```
+{
+  token(id:"0x1f9840a85d5af5bf1d1762f925bdaddc4201f984") {
+    symbol
+    name
+    decimals
+    volumeUSD
+    poolCount
+  }
+}
+```
+
+### Token Daily Aggregated
+
+You can fetch aggregate data about a specific token over a day. This query gets 10-day volume data for the UNI token ordered from oldest to newest.
```

Comment:
what do we mean "over a day"? if we mean over a 24hr-period, might make sense to say "24hr period" and then give an example of a 24hr period query rather than 10 days